docs/getting-started/bot.md

```diff
@@ -0,0 +1,412 @@
+Atomist can now be put to work, creating projects, editing code,
```

Comment:
That whole page must have made you suffer to write up. I feel for you :/

It feels nonetheless that it's quite hard to digest when reading it up as well. After a long trek through getting started, it feels even more words before getting hacking. What's your thoughts on that?

Would it make to make it leaner by removing most of the screenshots and texts.

I can't make my mind up. One the one hand, it's a very long page that is daunting to look at. On the other hand, I like the fact it describes the mechanics of Atomist and, for someone who reads it all, everything else would make much more sense.